File: Python/1934.py
## 최소공배수
import sys
input = sys.stdin.readline

t = int(input())

for _ in range(t):
    a, b = map(int, input().split())
    gcd = 1
    for i in range(min(a, b), 1, -1):
        if (a % i == 0) and (b % i == 0):
            gcd = i
            break
    print(int(a * b / gcd))

File: JAVA/src/_1934.java
import java.io.*;

// 최소공배수
public class _1934 {
    public static int gcd(int a, int b) {
        int c;
        while (b != 0) {
            c = a % b;
            a = b;
            b = c;
        }

        return a;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int t = Integer.parseInt(br.readLine());
        for (int i = 0; i < t; i++) {
            String[] input = br.readLine().split(" ");
            int a = Integer.parseInt(input[0]);
            int b = Integer.parseInt(input[1]);
            int gcd = gcd(a, b);
            bw.write(a * b / gcd + "\n");
        }

        bw.flush();
        bw.close();
    }
}
